libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -139,16 +142,20 @@ AP_GPS_SBF::read(void)
                                 }
                                 break;
                             case Config_State::Constellation:
-                                if ((params.gnss_mode&0x6F)!=0) {
-                                    //IMES not taken into account by Septentrio receivers
-                                    if (asprintf(&config_string, "sst, %s%s%s%s%s%s\n", (params.gnss_mode&(1U<<0))!=0 ? "GPS" : "",
-                                                            (params.gnss_mode&(1U<<1))!=0 ? ((params.gnss_mode&0x01)==0 ? "SBAS" : "+SBAS") : "",
-                                                            (params.gnss_mode&(1U<<2))!=0 ? ((params.gnss_mode&0x03)==0  ? "GALILEO" : "+GALILEO") : "",
-                                                            (params.gnss_mode&(1U<<3))!=0 ? ((params.gnss_mode&0x07)==0 ? "BEIDOU" : "+BEIDOU") : "",
-                                                            (params.gnss_mode&(1U<<5))!=0 ? ((params.gnss_mode&0x0F)==0 ? "QZSS" : "+QZSS") : "",
-                                                            (params.gnss_mode&(1U<<6))!=0 ? ((params.gnss_mode&0x2F)==0  ? "GLONASS" : "+GLONASS") : "") == -1) {
-                                        config_string=nullptr;
-                                    }
+                                //IMES not taken into account by Septentrio receivers
```

Comment:
Why is this commented out?

Reply:
It is part of the possible constellation choice in Mission Planner, I thought it would make it clear for future users and maintainers why it is not a possibility